src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import React, { useCallback, memo, type ComponentType } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { IconButton, Paper, MenuList, MenuItem, Divider } from '@material-ui/core';
+import { MoreHoriz } from '@material-ui/icons';
+import Popper from '../../Popper/Popper.component';
+import type { Props } from './listViewMenu.types';
+
+const getStyles = () => ({
+    subHeader: {
+        paddingLeft: 16,
+        paddingRight: 16,
+        paddingTop: 10,
+        paddingBottom: 10,
+        color: '#717c8b',
+        fontWeight: 500,
+        fontSize: 16,
+        '&:focus': {
+            outline: 'none',
+            color: 'black',
+        },
+    },
+    subHeaderDivider: {
+        '&:focus': {
+            outline: 'none',
+            backgroundColor: 'black',
+        },
+    },
+});
+
+const ListViewMenuPlain = ({ customMenuContents = [], classes }: Props) => {
+    const renderPopperAction = useCallback((togglePopper: Function) => (
+        <IconButton onClick={togglePopper}>
+            <MoreHoriz />
+        </IconButton>
+    ), []);
+
+    const renderMenuItems = useCallback((togglePopper: Function) =>
+        customMenuContents
+            .map((content) => {
+                if (content.subHeader) {
+                    return [
+                        <Divider
+                            key={`${content.key}divider`}
+                            data-test={`subheader-divider-${content.key}`}
+                            className={classes.subHeaderDivider}
+                        />,
+                        <div
+                            key={content.key}
+                            data-test={`subheader-${content.key}`}
+                            className={classes.subHeader}
+                        >
+                            {content.subHeader}
+                        </div>,
+                    ];
```

Comment:
Can we use react fragment instead of array in this case or is it a special case?

Reply:
Well I dont think one is better than the other. 

With this said, I think that in the other 95 percent of the code we are using fragment. So the question is what makes us wanna divert in this case and use an array?

Reply:
Honestly I don't remember why I made an array here, this is old code that was moved to new file. Seems to work as react fragment, changing.